src/client/controller/GameController.js

```diff
@@ -351,13 +355,28 @@ GameController.prototype.onBonusClear = function(e)
  *
  * @param {Event} e
  */
+GameController.prototype.onPosition = function(e)
+{
+    var avatar = this.game.avatars.getById(e.detail[0]),
+        position = this.compressor.decompressPosition(e.detail[1][0], e.detail[1][1]);
```

Comment:
You can compute position in the avatar condition, it won't be used if the condition is false anyway